core/transaction/entity.go

```diff
@@ -99,6 +99,8 @@ const (
 	STORAGESC_UPDATE_ALLOCATION        = "update_allocation_request"
 	STORAGESC_WRITE_POOL_LOCK          = "write_pool_lock"
 	STORAGESC_WRITE_POOL_UNLOCK        = "write_pool_unlock"
+	STORAGESC_ADD_CURATOR              = "add_curator"
+	STORAGESC_CURATOR_TRANSFER         = "curator_transfer_allocation"
```

Comment:
imho, we could drop "curator" suffix here and in similar cases (see below). to make it:
```
STORAGESC_TRANSFER_ALLOCATION = "transfer_allocation"
```
why? because "curator" doesn't play to be reflected in naming. it's just a condition or invisible subject in the operation where "transfer" is action and "allocation" is object. in other cases we just reflect actions and objects - "add_curator", for example.